site/src/components/page-faq/helpscout-webhook.ts

```diff
@@ -7,14 +8,18 @@ export const submitToHelpScout = async ( data: Object)
       status: number;
       data?: any;
    }> => {
+   const headers =  new Headers({
+      "Access-Control-Allow-Origin": "*",
+      "Access-Control-Allow-Methods": "GET,PUT,POST,DELETE,PATCH,OPTIONS",
+      "Access-Control-Allow-Headers": "Origin, X-Requested-With, Content-Type, Accept, Authorization",
+      "Content-Type": "application/json"
+   });
```

Comment:
These are the headers the server (Zapier endpoint) should be adding in response the the browser's pre-flight OPTION request. If it's possible to add them there, that should fix it.